inc/class-customizer-guided-tour.php

```diff
@@ -0,0 +1,213 @@
+							{{ data.altStep.buttonText }}
+						</a>
+					<# } #>
+					<# if ( data.buttonText ) { #>
+						<a class="sf-nux-button sf-nux-primary-button {{ data.className }}" href="#">
+							{{ data.buttonText }}
+						</a>
+					<# } #>
+
+					<# if ( data.showSkip ) { #>
+						<a class="sf-guided-tour-skip" href="#">
+							<?php esc_attr_e( 'Skip this step', 'wc-api-dev' ); ?>
+						</a>
+					<# } #>
+				</div>
+			</script>
+			<?php
+		}
+
+		/**
+		 * Guided tour steps.
+		 *
+		 * @since 0.8.7
+		 */
+		public function guided_tour_steps() {
+			$steps = array();
+
+			$steps[] = array(
+				'title'       => __( 'Customize Your Store', 'storefront' ),
+				'message'     => __( 'It looks like your current theme isn\'t ready for shop features yet - your shop pages might look a little funny.</p><p>We suggest switching themes to <b>Storefront</b> which will bring out the best in your shop. Don\'t worry, if you try Storefront now, it won\'t be activated until you save your changes in the Customizer', 'storefront' ),
+				'buttonText'  => __( 'I\'ll keep my current theme', 'storefront' ),
+				'section'     => '#customize-info',
+				'className'   => 'sf-button-secondary',
+				'altStep'     => array(
+					'buttonText'  => __( 'I\'ll try Storefront!', 'storefront' ),
+					'action'      => 'expandThemes',
+					'message'     => __( 'Click the thumbnail to get started with Storefront', 'storefront' ),
+					'section'     => '#customize-control-theme_storefront .theme-screenshot',
+				)
+			);
+
+			// Determine what the next step should be
+			$needsLogo = ! has_custom_logo() && current_theme_supports( 'custom-logo' );
+			$logoBullet = $needsLogo ? __( '<li>Add your logo</li>', 'storefront' ) : '';
+
+			$steps[] = array(
+				'message'     => __( 'Okay! Remember you can switch themes at any time.</p><p>To get your store looking great, let\'s run through some common tasks:</p><ul>' . $logoBullet . '<li>Add Shop pages to your menus</li><li>Set your shop page as the homepage</li></ul><p>', 'storefront' ),
+				'buttonText' => $needsLogo ? __( 'Add a logo', 'storefront' ) : __( 'Add menu items', 'storefront' ),
+				'section'     => '#accordion-section-themes',
+				'altStep'    => array(
+					'buttonText' => __( 'I\'ll figure it out for myself', 'storefront' ),
+					'className'   => 'sf-button-secondary',
+					'action'      => 'exit',
+				)
+			);
+
+			if ( $needsLogo ) {
+				$steps[] = array(
+					'title'       => __( 'Add your logo', 'storefront' ),
+					'action'      => 'addLogo',
+					'message'     => __( 'Click the \'Select Logo\' button to upload your logo. After you upload your logo, click next to update your menus.', 'storefront' ),
+					'buttonText'  => __( 'Next', 'storefront' ),
+					'section'     => 'title_tagline',
+				);
+			}
+
+			$steps[] = array(
+				'message'      => __( 'Choose a menu to add shop pages to', 'storefront' ),
```

Comment:
This step could use a copy-review, it's very short and feels like it's missing something (even just a . at the end, but I think it needs to be reworded).

![](https://user-images.githubusercontent.com/541093/31023132-ebbca8ae-a508-11e7-9940-e46dda6ae5a2.png)

Reply:
@jameskoster ^ any ideas on updated copy for this?